templates/0000-template.md

```diff
@@ -0,0 +1,92 @@
+- Feature Name: (fill me in with a unique ident, `my_awesome_feature`)
+- Start Date: (fill me in with today's date, YYYY-MM-DD)
+
+# Summary
+[summary]: #summary
+
+One paragraph explanation of the feature.
+
+# Motivation
+[motivation]: #motivation
+
+Why are we doing this? What use cases does it support? What is the expected outcome?
+
+# Guide-level explanation
+[guide-level-explanation]: #guide-level-explanation
+
+Explain the proposal as if it were already included in the PartiQL specification or public APIs. The explanation should assume
+the reader has proficient knowledge on the existing PartiQL specification.
+
+- Introducing new named concepts.
+- Explaining the feature largely in terms of examples.
+- Explaining how PartiQL users should *think* about the feature, and how it should impact the way they use PartiQL. It should explain the impact as concretely as possible.
+- If applicable, provide sample error messages, deprecation warnings, or migration guidance.
+- If applicable, describe the differences between teaching this to existing PartiQL users and new PartiQL users.
+
+For implementation-oriented RFCs, this section should focus on how PartiQL contributors should think about the change, and give examples of its concrete impact.
+For api-oriented RFCs, this section should focus on how PartiQL users would leverage or be affect by the changes.
+
+# Reference-level explanation
+[reference-level-explanation]: #reference-level-explanation
+
+This is the technical portion of the RFC. Explain the design in sufficient detail that:
+
+- Its interaction with other features is clear.
+- It is reasonably clear how the feature would be implemented.
+- Corner cases are dissected by example.
+
+The section should return to the examples given in the previous section, and explain more fully how the detailed proposal makes those examples work.
+
+# Drawbacks
+[drawbacks]: #drawbacks
+
+Why should we *not* do this?
+
+# Rationale and alternatives
+[rationale-and-alternatives]: #rationale-and-alternatives
+
+- Why is this design the best in the space of possible designs?
+- What other designs have been considered and what is the rationale for not choosing them?
+- What is the impact of not doing this?
+
+# Prior art
+[prior-art]: #prior-art
+
+Discuss prior art, both the good and the bad, in relation to this proposal.
+A few examples of what this can include are:
+
+- For specification proposals: Does this feature exist in SQL-92 or other SQL dialects?
```

Comment:
Suggestion: `SQL-92 -> any ISO SQL standard` b/c each ISO SQL standard replaces the previous one.